src/main/java/org/fcrepo/migration/handlers/ocfl/ArchiveGroupHandler.java

```diff
@@ -148,68 +137,57 @@ public void processObjectVersions(final Iterable<ObjectVersionReference> version
                 final String dsId = dv.getDatastreamInfo().getDatastreamId();
                 final String f6DsId = resolveF6DatastreamId(dsId, f6ObjectId, mimeType);
```

Comment:
I think it is better to use fedoraDsId rather than embedding the version in the variable name unless you have an objection.

Reply:
Same with the resolveF6... method name.